lib/process-services-cloud/src/lib/app/components/app-details-cloud.component.spec.ts

```diff
@@ -55,4 +56,15 @@ describe('AppDetailsCloudComponent', () => {
         app.click();
         expect(component.selectedApp.emit).toHaveBeenCalledWith(fakeApplicationInstance[0]);
     });
+
+    it('should render card with default icon and theme when are not provided', () => {
```

Comment:
this test is not failing in development means that you are not testing the right scenario. Please write your unit test before the code and check that they fail before to add your fix/feature